pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/PersistentDispatcherSingleActiveConsumer.java

```diff
@@ -543,19 +543,13 @@ public void updateRateLimiter(DispatchRate dispatchRate) {
         if (!this.dispatchRateLimiter.isPresent() && dispatchRate != null) {
```

Comment:
We can call `initializeDispatchRateLimiterIfNeeded` here after remove the params.